index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Sanity Checklist Prototype</title>
+        <link rel="stylesheet" href="styles/main.css">
+        <link rel="stylesheet" href="styles/main-header.css">
+        <link rel="stylesheet" href="styles/main-checklist.css">
+        <link rel="stylesheet" href="styles/main-options.css">
+    </head>
+
+    <body>
+        <header class="fix-top">
+            <h1 style="text-align: center;">OOTxMM Randomizer - Sanity-Checklist</h1>
+        </header>
+
+        <main>
+            <section class="display-options">
+                <input type="checkbox"> Hide completed categories
+                <input type="checkbox"> Hide completed checks
+            </section>
+
+            <section class="sanity-options">
+                <div class="game">
+                    <div>
+                        <label>OoT</label>
+                        <input type="radio" name="game" checked>
+                    </div>
+                    
+                    <div>
+                        <label>MM</label>
+                        <input type="radio" name="game">
+                    </div>
+                </div>
+
+                <div class="area-sanities">
+                    <div class="sanity">
+                        <div class="drop-name">
+                            <label for="sanity">Sanity Type:</label>
+                            <div class="hint">
+                                <label>?</label>
+                                <div class="tooltip">
+                                    <p><strong>Scrubs: </strong>Each Deku Scrub that do a one-time trade</p>
+                                    <p><strong>Shops: </strong>Each Shop has a total of 8 completely random items</p>
+                                    <p><strong>Cows: </strong>Each Shop will provide a random Item when <strong>Epona&apos;s Song</strong> has been played</p>

```

Comment:
small typo, should read "Each Cow" instead of shop

Reply:
oh bruh, I did not notice
will change it with the next update

Reply:
i'm merging this in now, i have the main branch backed up to a new branch and set that one to the default branch for the repo. once that is done, ill rename the branches again